nds-org/workbench-apiserver-golang
# Fix ndslabsctl add account

## Problem
Add account creates users with a blank `status`.

Also, our golang imports/package names need to be updated to point at this new repo.

## Approach
* Set `status` to approved when using the admin-only `ndslabsctl add account` function
* Fix all of our locally-imported packages to point at new repo

## How to Test
1. Deploy the Helm chart with `workbench.images.apiserver=ndslabs/apiserver:fix-ndslabsctl-add-account`
2. Fetch the "admin" user password: `kubectl exec -it deploy/workbench -c apiserver -- cat password.txt`
    * Copy this password to your clipboard
3. Login as the admin user with this password: `kubectl exec -it deploy/workbench -c apiserver -- ndslabsctl login admin`
    * Paste the password from your clipboard
    * You should see "Login succeeded"
4. Add the demo account via file: `kubectl exec -it deploy/workbench -c apiserver -- ndslabsctl add account -f /templates/demo-account.json`
     * You should see "Added account demo"
     * You should be able to log into the Workbench UI with the credentials `demo:demo123`
5. Add a new account via CLI parameters: `kubectl exec -it deploy/workbench -c apiserver -- ndslabsctl add account testuser asdf1234`
     * You should see "Added account testuser"
     * You should be able to log into the Workbench UI with the credentials `testuser:asdf1234`
